README.md

```diff
@@ -226,6 +231,21 @@ WHERE rds.storage_encrypted = false
 return a.name as AWSAccount, count(rds) as UnencryptedInstances
 ```
```

Comment:
Use `####` not `###` to match indent level with the other example questions in this section.

Reply:
fixed